jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
What traceId should be used in a newly created span when there are references from different traces?

Create new traceId as a "join" or make an assumption that `childOf` is "stronger" and newly created span should inherit traceId?